tools/hangar/__snapshots__/invalid.ts.snap

```diff
@@ -1069,9 +1077,6 @@ error: Expected 'b' to be a type but it's a variable
   |                          ^ Expected 'b' to be a type but it's a variable
 
 
-error: Inflight class c's parent \\"any\\" is not a class
```

Comment:
This error and the one below are are removed since we already give an error in the same place ("Expected 'b' to be a type but it's a variable")